tests/test_core.py

```diff
@@ -1046,7 +1050,7 @@ def _repeat_instantiation(event):
 
 
 def test_scaper_instantiate():
-    for sr in (44100, 22050):
+    for sr in SAMPLE_RATES:
```

Comment:
Good refactor. We should some day look at refactoring tests a bit more but this is a good start.